lib/repl.js

```diff
@@ -794,7 +794,16 @@ function complete(line, callback) {
     filter = match[1];
     var dir, files, f, name, base, ext, abs, subfiles, s;
     group = [];
-    var paths = module.paths.concat(require('module').globalPaths);
+    let paths = [];
+    if (completeOn === '.') {
+      group = ['./', '../'];
```

Comment:
@TimothyGu wait, no, it shouldn't :) One can only require them as `./.filename`, and they will be completed on `require('./`.